app/stac_api/admin.py

```diff
@@ -15,14 +15,11 @@
 from django.http import HttpResponseRedirect
 from django.urls import reverse
 
-from solo.admin import SingletonModelAdmin
```

Comment:
was this the only we used that `SingletonModelAdmin`? If yes we could remove the package from the dependancy list.

Reply:
Thanks for the hint, I removed the dependency. :+1: